pkg/operator/ceph/object/objectstore.go

```diff
@@ -107,7 +107,7 @@ func removeObjectStoreFromMultisite(objContext *Context, spec cephv1.ObjectStore
 	zoneEndpoints := strings.Join(zoneEndpointsList, ",")
 	endpointArg := fmt.Sprintf("--endpoints=%s", zoneEndpoints)
 
-	zoneIsMaster, err := checkZoneIsMaster(objContext)
+	zoneIsMaster, _ := checkZoneIsMaster(objContext)
```

Comment:
ditto do not silence the error, handle it

Reply:
Done.